quinn/src/streams.rs

```diff
@@ -770,7 +781,7 @@ pub enum ReadExactError {
     FinishedEarly,
     /// A read error occurred
     #[error("{0}")]
-    ReadError(#[source] ReadError),
+    ReadError(#[from] ReadError),
```

Comment:
Should this refactoring be separated out?

Reply:
Yup, done.